packages/cli/src/__tests__/fix.ts

```diff
@@ -70,6 +70,59 @@ test("set main and module field", async () => {
   `);
 });
 
+test("set exports field when opt-in", async () => {
+  let tmpPath = f.copy("package-exports");
+
+  await modifyPkg(tmpPath, (json) => {
+    json.exports = { ".": { browser: {}, worker: {} } };
```

Comment:
q: why this is needed? shouldn't a base `json.exports` be generated solely based on `json.preconstruct.exports`? It seems that right now both are required.

Reply:
This used to be they way you opt into the feature (and provide which conditions you wanted to support). I've removed that requirement and now you can specify which conditionals you want to support by passing an array of condition names to the `exports` field in the preconstruct config for the package.